indico/modules/events/abstracts/templates/management/notification_tpl_form.html

```diff
@@ -1,10 +1,9 @@
 {% from 'forms/_form.html' import form_header, form_footer, form_rows, form_row, form_fieldset %}
 
 {{ form_header(form) }}
-    {{ form_row(form.title) }}
+    {{ form_rows(form, fields=('title', 'stop_on_match')) }}
```

Comment:
`{{ form_rows(form, skip=('rules','default_tpl')) }}` works fine.  

Know this shouldn't it be more appropriate using `{{ form_rows(form, fields=('title','stop_on_match')) }}`?